.env

```diff
@@ -0,0 +1,7 @@
+# build/release/Makefile
```

Comment:
Aren't dot env files usually secret? i.e. not checked into git

Reply:
Also, this is not the format of a dot env file. I think you meant to create a .sh?

Reply:
@zalsader yes, but we are using it as a workaround to use Rook upstream code itself for a release without the trouble of maintenance https://github.com/rook/rook/pull/11968/files#diff-bff298d6fabc712b5a480b11ab34e9de4d983b2c50e81f3f8c211e6a54ad559dR37 

I do not see any other way of how we can avoid .env if we want to be compatible with rook upstream, let's discuss this in watercooler maybe.

Reply:
Not including `export` in the `.env` file and using this instead of `source` is a better approach:
```sh
export $(grep -v '^#' .env | xargs -d '\n')
```
or even
```sh
set -o allexport
source conf-file
set +o allexport
```
[See this answer.](https://stackoverflow.com/a/20909045/7557028)